services/src/pro/contexts/schema.sql

```diff
@@ -1,22 +1,67 @@
+CREATE TYPE "Band" AS (
```

Comment:
maybe call it StackBand?

Reply:
[3320d8f](https://github.com/geo-engine/geoengine/pull/866/commits/3320d8fb10c3b7900e07ef0a20c5cfd59ef38955)